internal/formatters/fmt_multi.go

```diff
@@ -34,6 +34,18 @@ func (r repeater) SetStorage(s *storage.Storage) {
 	}
 }
 
+// TestRunStarted triggers TestRunStarted for all added formatters.
+func (r repeater) Close() error {
+	var err error
+	for _, f := range r {
+		e := f.Close()
+		if e != nil {
+			err = e
+		}
+	}
+	return err
+}
+
```

Comment:
You forgot to update the comment. Also do we need to add test for this method?

Reply:
Will see what the existing test strat is for this and look to improve.